proto/ibc/client/client.proto

```diff
@@ -15,13 +15,22 @@ message IdentifiedClientState {
   google.protobuf.Any client_state = 2 [(gogoproto.moretags) = "yaml:\"client_state\""];
 }
 
+// ConsensusStateWithHeight defines a consensus state with an additional height field.
+message ConsensusStateWithHeight {
```

Comment:
I am not a huge fan of this name. Any other ideas?

Reply:
I think this is fine IMHO